PR: https://github.com/odoo/odoo/pull/

From: 09e380e239e74835aa8fc5ea75e39eaecacdf75b
From: Nicolas Martinelli
Date: 2017-10-03 10:48:48

Structural Changes: 2
Total Changes: 32

[FIX] base, website_customer: incorrect wording

opw-768886

================================= pseudo patch: =================================

--- a/odoo/addons/base/i18n/base.pot
+++ b/odoo/addons/base/i18n/base.pot
@@ -6487,13 +6487,13 @@ msgid "Click to create a new bank."
 msgstr ""
 
 #. module: base
-#: model:ir.actions.act_window,help:base.action_partner_category_form
-msgid "Click to create a new partner tag."
+#: model:ir.actions.act_window,help:base.res_partner_industry_action
+msgid "Click to create a new industry."
 msgstr ""
 
 #. module: base
-#: model:ir.actions.act_window,help:base.res_partner_industry_action
-msgid "Click to create a new sector of activity."
+#: model:ir.actions.act_window,help:base.action_partner_category_form
+msgid "Click to create a new partner tag."
 msgstr ""
 
 #. module: base
@@ -10273,6 +10273,12 @@ msgstr ""
 msgid "Indonesia"
 msgstr ""
 
+#. module: base
+#: model:ir.model.fields,field_description:base.field_res_partner_industry_id
+#: model:ir.model.fields,field_description:base.field_res_users_industry_id
+msgid "Industry"
+msgstr ""
+
 #. module: base
 #: model:ir.model.fields,field_description:base.field_ir_model_info
 #: model:ir.ui.view,arch_db:base.module_form

--- a/odoo/addons/base/res/res_partner.py
+++ b/odoo/addons/base/res/res_partner.py
@@ -196,7 +196,7 @@ class Partner(models.Model):
     mobile = fields.Char()
     is_company = fields.Boolean(string='Is a Company', default=False,
         help="Check if the contact is a company, otherwise it is a person")
-    industry_id = fields.Many2one('res.partner.industry', 'Sector of Activity')
+    industry_id = fields.Many2one('res.partner.industry', 'Industry')
     # company_type is only an interface field, do not use it in business logic
     company_type = fields.Selection(string='Company Type',
         selection=[('person', 'Individual'), ('company', 'Company')],
@@ -777,7 +777,7 @@ class Partner(models.Model):
 
 
 class ResPartnerIndustry(models.Model):
-    _description = 'Sector of Activity'
+    _description = 'Industry'
     _name = "res.partner.industry"
     _order = "name"
 

--- a/odoo/addons/base/res/res_partner_view.xml
+++ b/odoo/addons/base/res/res_partner_view.xml
@@ -662,12 +662,12 @@
             </field>
         </record>
 
-        <!-- Sector of Activity -->
+        <!-- Industry -->
         <record id="res_partner_industry_view_form" model="ir.ui.view">
-            <field name="name">Sector of Activity</field>
+            <field name="name">Industry</field>
             <field name="model">res.partner.industry</field>
             <field name="arch" type="xml">
-                <form string="Sector of Activity">
+                <form string="Industry">
                     <group col="4">
                         <field name="name"/>
                         <field name="full_name"/>
@@ -677,11 +677,11 @@
             </field>
         </record>
         <record id="res_partner_industry_view_tree" model="ir.ui.view">
-            <field name="name">Sector of Activity</field>
+            <field name="name">Industry</field>
             <field name="model">res.partner.industry</field>
             <field eval="6" name="priority"/>
             <field name="arch" type="xml">
-                <tree string="Sector of Activity">
+                <tree string="Industry">
                     <field name="name"/>
                     <field name="full_name"/>
                     <field name="active"/>
@@ -689,14 +689,14 @@
             </field>
         </record>
         <record id="res_partner_industry_action" model="ir.actions.act_window">
-            <field name="name">Sector of Activity</field>
+            <field name="name">Industry</field>
             <field name="type">ir.actions.act_window</field>
             <field name="res_model">res.partner.industry</field>
             <field name="view_type">form</field>
             <field name="view_mode">tree,form</field>
             <field name="help" type="html">
                 <p class="oe_view_nocontent_create">
-                    Click to create a new sector of activity.
+                    Click to create a new industry.
                 </p>
                 <p>
                     Manage sectors of activity to better classify partners for tracking and analysis purposes.
